### PR TITLE
fix(ios): Updated selector for log

### PR DESCRIPTION
**Pull request by vijaysingh-axway**
https://jira.appcelerator.org/browse/TIMOB-27055

This PR fixes 3 tickets- 
1. [TIMOB-27054](https://jira.appcelerator.org/browse/TIMOB-27054)
2.[ TIMOB-27055](https://jira.appcelerator.org/browse/TIMOB-27055)
3.[TIMOB-27056](https://jira.appcelerator.org/browse/TIMOB-27056)

### CODE REVIEW

File: iphone/Classes/TiUIWebView.m [hunk 1]
```diff
@@ -387,7 +387,7 @@ - (WKUserScript *)userScriptTitaniumInjectionForAppEvent
     [(TiUIWebViewProxy *)self.proxy setPageToken:_pageToken];
   }
 
-  NSString *source = @"var callbacks = {}; var Ti = {}; Ti.pageToken = %@; \
+  NSString *source = @"var callbacks = {}; var Ti = {}; var Titanium = {}; Ti.pageToken = %@; \
```
Comment (jquick-axway) [inline iphone/Classes/TiUIWebView.m]: Wouldn't it be easier to do this instead?
```javascript
var Titanium = Ti;
```

That way you don't have to duplicate the API definitions for the `Titanium` object down below. Both `Ti` and `Titanium` will reference the same object.

**Review by janvennemann — CHANGES_REQUESTED** [review]
As Josh already mentioned, avoid the duplicate code and just reference the same object for `Ti` and `Titanium`.

File: iphone/Classes/TiUIWebView.m (hunk 1 above)
Comment (vijaysingh-axway) [reply iphone/Classes/TiUIWebView.m]: Yeah. You are right. Updated the PR. Thanks!

**Review by sgtcoolguy — APPROVED** [review]
LGTM

**Comment by keerthi1032** [comment]
FR Passed. logs shown as expected in console .
Test Environment:
  Name                        = Mac OS X
  Version                     = 10.13.6
  Architecture                = 64bit
Node.js
  Node.js Version             = 8.9.1
  npm Version                 = 5.5.1
Titanium CLI
  CLI Version                 = 5.1.1
Titanium SDK
  SDK Version                 = local sdk 8.1.0.v20190531134952 and local sdk 8.0.2.v20190530104542
CLi =7.0.11
Device =iPhone 6s ios12 
Simulator =iphone xr ios12

**Comment by keerthi1032** [comment]
Jenkin is failing .not able to merge. waiting for resolving jenkin's failure.

**Pull request merged**